### ISSUE TITLE
Implement simple output

### ISSUE THREAD

**Issue by timspainNERSC** [nextsimdg/nextsimdg#100]
Based on the ModelComponent code base, implement simple output in a modular fashion.

- Implement the code to actual get the data for most components
- Class(es) to control the timing and volume of data
- A simple implementation of the output interface which directly writes a netCDF file in the most bone-headed simple way.

**Comment by timspainNERSC** [comment]
The output specification needs to be configured.
A special section of the config file 
`[Output]`
Wherein the output specifications are configured by `fieldname.start = `_timepoint_, `fieldname.step = `_duration_ and `fieldname.stop = `_timepoint_.
Assuming a YYYYMMDDThhmmss time format, an example output section in the config file might look like:
```
[Output]
hice.start = 2022-05-25T112500
hice.step = 5 min
hice.stop = 2022-05-25T115500
cice.start = 2022-05-25T112500
cice.step = 1 day
cice.stop = 2022-05-30T112500
```

**Comment by timspainNERSC** [comment]
Depends on issue #102

**Comment by timspainNERSC** [comment]
Closed as part of PR #101.
